ray-operator/test/e2e/stop.py

```diff
@@ -0,0 +1,12 @@
+import time
+from ray.job_submission import JobSubmissionClient
+
+print("Sleep 20 seconds to enable the RayJob to transition to RUNNING state")
+time.sleep(20)
```

Comment:
Could we pass in submission ID into stop.py and poll the job status until it's RUNNING?

Reply:
I agree. Or could we even delete the sleep entirely? Isn't it be guaranteed to be RUNNING at the time this Python code is being executed?

Reply:
RUNNING here refers to JobDeploymentStatus rather than JobStatus.

>  Isn't it be guaranteed to be RUNNING at the time this Python code is being executed?

`JobStatus` guaranteed to be RUNNING at the time this Python code is being executed. That's why I said "RayJob transition to RUNNING state" instead of "Ray job".

Reply:
I see, thanks for clarifying. Sleeping in a test isn't ideal but I don't know a better way right now

Reply:
Yes, we don't have a method to know the JobDeploymentStatus without implementing some RBAC configurations.